users/templates/base.html

```diff
@@ -39,98 +40,64 @@
             <li>
-                {% if request.path|startswith:custom_users_notifications %}
-                <a class="active" href="{{ custom_users_notifications }}">
-                    <img src="{% static 'users/images/white/notifications.png' %}">
-                    <span>{% trans 'Notifications' %}</span>
+                <a href="{{ custom_users_notifications }}" {% if request.path|startswith:custom_users_notifications %}
+                   class="active"{% endif %}>
+                    <span class="link-icon"><i class="fa fa-bell"></i></span>
+                    <span class="link-text">{% trans 'Notifications' %}</span>
                 </a>
-                {% else %}
-                <a href="{{ custom_users_notifications }}">
-                    <img src="{% static 'users/images/grey/notifications.png' %}">
-                    <span>{% trans 'Notifications' %}</span>
-                </a>
-                {% endif %}
             </li>
             <li>
-                {% if request.path|startswith:custom_projects_list_url or request.path|startswith:'/employees/reports/project/' %}
-                <a class="active" href="{{ custom_projects_list_url }}">
-                    <img src="{% static 'users/images/white/iko-04.png' %}">
-                    <span>{% trans 'Projects' %}</span>
-                </a>
-                {% else %}
-                <a href="{{ custom_projects_list_url }}">
-                    <img src="{% static 'users/images/grey/ico-04.png' %}">
-                    <span>{% trans 'Projects' %}</span>
+                <a href="{{ custom_projects_list_url }}"
+                   {% if request.path|startswith:custom_projects_list_url or request.path|startswith:'/employees/reports/project/' %}
+                    class="active"{% endif %}>
+                    <span class="link-icon"><i class="fa fa-cubes"></i></span>
+                    <span class="link-text">{% trans 'Projects' %}</span>
                 </a>
-                {% endif %}
             </li>
             {% endif %}
             <li>
-                {% if request.path == custom_report_list_url or request.path|startswith:custom_report_list_actual_date_url %}
-                <a class="active" href="{{ custom_report_list_url }}">
-                    <img src="{% static 'users/images/white/iko-05.png' %}">
-                    <span>{% trans 'Reports' %}</span>
-                </a>
-                {% else %}
-                <a href="{{ custom_report_list_url }}">
-                    <img src="{% static 'users/images/grey/ico-05.png' %}">
-                    <span>{% trans 'Reports' %}</span>
+                <a href="{{ custom_report_list_url }}"
+                   {% if request.path == custom_report_list_url or request.path|startswith:custom_report_list_actual_date_url %}
+                   class="active"{% endif %}>
+                    <span class="link-icon"><i class="fa fa-clipboard"></i></span>
```

Comment:
I suggest a little bigger image
```suggestion
                    <span class="link-icon"><i class="fa fa-clipboard fa-2x"></i></span>
```